googletest/src/matcher_support/edit_distance.rs

```diff
@@ -19,7 +19,7 @@ use std::fmt::Debug;
 ///
 /// Increasing this limit increases the accuracy of [`edit_list`] while
 /// quadratically increasing its worst-case runtime.
-const MAX_DISTANCE: i32 = 25;
+const MAX_DISTANCE: i32 = 50;
```

Comment:
I was probably far too conservative in setting this value. It should be finite to avoid potential quadratic time complexity in computing the diff, but feel free to raise it even more.